### PR TITLE
Feature/paselc 579

### PR DESCRIPTION
**Pull request by GiacomoBrancazi**
#### List of Changes

add new api for get tavolo operativo

#### Motivation and Context

add new api for get tavolo operativo

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.api.tavoloOp.getTavoloOpDetails}")
+    public TavoloOpResource getTavoloOpDetails(@ApiParam("${swagger.request.ecCode}")
+                                               @PathVariable("ecCode") String ecCode) {
+
+        TavoloOpOperations tavoloOpOperations = tavoloOpService.findByTaxCode(ecCode);
+        return TavoloOpMapper.toResource(tavoloOpOperations);
+    }
+
+    @PostMapping(value = "/create", consumes = MediaType.APPLICATION_JSON_VALUE)
```
Comment (jacopocarlini) [inline web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java:43]:
```suggestion
    @PostMapping(value = "/", consumes = MediaType.APPLICATION_JSON_VALUE)
```

File: web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping(value = "/tavoloop", produces = MediaType.APPLICATION_JSON_VALUE)
+@Api(tags = "tavoloop")
```
Comment (jacopocarlini) [inline web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java:21]:
```suggestion
@RequestMapping(value = "/tavolo-operativo", produces = MediaType.APPLICATION_JSON_VALUE)
@Api(tags = "Tavolo Operativo")
```

File: connector-api/src/test/java/it/pagopa/selfcare/pagopa/backoffice/connector/model/DummyTavoloOpEntity.java
Comment (jacopocarlini) [inline connector-api/src/test/java/it/pagopa/selfcare/pagopa/backoffice/connector/model/DummyTavoloOpEntity.java:1]: is this a relevant name? 
`DummyTavoloOpEntity.java`

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
it's ok, I just leave some cosmetics tips

File: web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java (hunk 2 above)
Comment (GiacomoBrancazi) [reply web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/TavoloOpController.java]: fixed

File: connector-api/src/test/java/it/pagopa/selfcare/pagopa/backoffice/connector/model/DummyTavoloOpEntity.java
Comment (jacopocarlini) [reply connector-api/src/test/java/it/pagopa/selfcare/pagopa/backoffice/connector/model/DummyTavoloOpEntity.java:1]: used for testing

**Review by jacopocarlini — APPROVED** [review]
Good work!

**Pull request merged**